e2e/utils/test-utils.ts

```diff
@@ -16,8 +16,10 @@ import WorkspacesPage from 'app/page/workspaces-page';
 import Navigation, { NavLink } from 'app/component/navigation';
 import { makeWorkspaceName } from './str-utils';
 import { config } from 'resources/workbench-config';
+import { logger } from 'libs/logger';
 
 export async function signIn(page: Page, userId?: string, passwd?: string): Promise<void> {
+  logger.info(`Sign in with Google to Workbench application`);
```

Comment:
Help to identify start of a test in test log when test file contains more than 1 test.